internal/messaging/service.go

```diff
@@ -40,6 +40,7 @@ var (
 	_ Service = (*seatMapService)(nil)
 	_ Service = (*seatMapAvailabilityService)(nil)
 	_ Service = (*seatMapAvailabilityService)(nil)
```

Comment:
is seatMapAvailabilityService duplicated?

Reply:
removed